GeneratingDataset.py

```diff
@@ -2651,21 +2652,25 @@ def __init__(self, path, audio, targets,
     self._seq_order = None  # type: typing.Optional[typing.List[int]]
     self.init_seq_order()
 
-  def _read(self, filename):
+  def _read(self, filename, zip_index):
     """
     :param str filename: in zip-file
+    :param int zip_index: index of the zip file to load, unused when loading without zip
     :rtype: bytes
     """
-    if self._zip_file is not None:
-      return self._zip_file.read(filename)
-    return open("%s/%s" % (self.path, filename), "rb").read()
+    if self._zip_files is not None:
+      return self._zip_files[zip_index].read(filename)
+    return open("%s/%s" % (self.paths[0], filename), "rb").read()
 
-  def _collect_data(self):
+  def _collect_data_part(self, zip_index):
     """
-    :return: entries
+    collect all the entries of a single zip-file or txt file
+    :param int zip_index: index of the zip-file in self._zip_files, unused when loading without zip
+    :return: data entries
     :rtype: list[dict[str]]
     """
-    data = eval(self._read("%s.txt" % self._name))  # type: typing.List[typing.Dict[str]]
+    import time
```

Comment:
Remove unused imports (check PyCharm warnings).

Reply:
unused import is unfortunately not displayed as a warning in my case, I am still searching for where to enable it.